.vscode/settings.json

```diff
@@ -163,6 +164,7 @@
       "rootPath": "packages/aila"
     }
   ],
+  "jest.runMode": "on-demand",
```

Comment:
I need this because otherwise tests auto-run on save.

Reply:
For what it's worth, I use that functionality when working on unit tests. Is there a way to opt long-running tests out of it?

Reply:
That said, personally I've changed editor away from vscode. Unless you're touching a Zed config file I'm all good!